applications/asset_tracker/src/motion/motion.h

```diff
@@ -43,12 +43,12 @@ typedef struct {
 	double x;			/**< X-axis acceleration [m/s^2]. */
 	double y;			/**< y-axis acceleration [m/s^2]. */
 	double z;			/**< z-axis acceleration [m/s^2]. */
+	int64_t ts;			/**< Data sampling uptime. */
```

Comment:
Wouldn't `motion_data_t` be a better place for timestamp than `motion_acceleration_data_t`? It's valid for both, acceleration and orientation.

Reply:
Agreed.